gemhardware/src/common/amc13/AMC13Manager.cc

```diff
@@ -307,22 +307,26 @@ void gem::hw::amc13::AMC13Manager::initializeAction()
   gem::utils::LockGuard<gem::utils::Lock> guardedLock(m_amc13Lock);
 
   // enable daq link (if SFP mask is non-zero
-  DEBUG("Enabling DAQLink with settings: fake data:" << m_enableFakeData
+  DEBUG("AMC13Manager::initializeAction Enabling DAQLink with settings: fake data:" << m_enableFakeData
         << ", sfpMask:" << m_sfpMask);
 
   p_amc13->fakeDataEnable(m_enableFakeData);
   p_amc13->daqLinkEnable(m_enableDAQLink);
 
   // enable SFP outputs based on mask configuration
   p_amc13->sfpOutputEnable(m_sfpMask);
-  /*
+
   // ignore AMC tts state per mask
-  if (m_ignoreAmcTts) {
-    p_amc13->write(amc13::AMC13Simple::T1, "CONF.AMC.TTS_DISABLE_MASK", 0xfff);
+  INFO("AMC13Manager::initializeAction m_amcIgnoreTTSList " << m_amcIgnoreTTSList);
+  m_ignoreAMCTTS = p_amc13->parseInputEnableList(m_amcIgnoreTTSList,true);
+  INFO("AMC13Manager::initializeAction m_amcIgnoreTTSList " << m_amcIgnoreTTSList
```

Comment:
Do we really want these two printouts as INFO? I would remove the line 320

Reply:
Good catch, I think the second should be `m_ignoreAMCTTS`

Reply:
New implementation should now be pushed